packages/js/tailwind.config.js

```diff
@@ -0,0 +1,46 @@
+function defaultColor(baseName) {
+  return `var(--${baseName})`;
+}
+
+function generateColorShades(baseName) {
+  return {
+    50: `var(--${baseName}-50)`,
+    100: `var(--${baseName}-100)`,
+    200: `var(--${baseName}-200)`,
+    300: `var(--${baseName}-300)`,
+    400: `var(--${baseName}-400)`,
+    500: `var(--${baseName}-500)`,
+    600: `var(--${baseName}-600)`,
+    700: `var(--${baseName}-700)`,
+    800: `var(--${baseName}-800)`,
+    900: `var(--${baseName}-900)`,
+  };
+}
+
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ['./src/**/*.{js,jsx,ts,tsx}'],
+  prefix: 'nt-',
+  corePlugins: {
+    preflight: false,
+  },
+  theme: {
+    extend: {
+      colors: {
+        primary: { DEFAULT: defaultColor('nv-color-primary'), ...generateColorShades('nv-color-primary') },
```

Comment:
💭 **thought (non-blocking):** the `DEFAULT` could be a part of the `generateColorShades` function

Reply:
We don't want a default for every usage of that function, just because we generate solid AND alpha shades, and that would mean two DEFAULTs.